components/createNft/SellingOptionsSection.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ChangeEvent, Dispatch, SetStateAction, useState } from 'react';
+import HbarLogo from '../../public/svg/HbarLogo';
+import classNames from '../../utils/classNames';
+import { sellingOptions } from '../../utils/Constants';
+import {
+  NftForm,
+  NftFormErrors,
+  NftInCollection,
+  SellingOption,
+  StepTwoErrors,
+} from '../../utils/Interfaces';
+import ErrorMessage from '../common/ErrorMessage';
+import Input from '../common/Input';
+
+interface SellingOptionsSectionProps {
+  formData: NftForm | NftInCollection;
+  handleFormChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  setFormData: Dispatch<SetStateAction<NftForm | NftInCollection>>;
+  formDataErrors: NftFormErrors | StepTwoErrors;
+  setFormDataErrors: Dispatch<SetStateAction<NftFormErrors | StepTwoErrors>>;
+}
+
+const SellingOptionsSection = function ({
+  formData,
+  setFormData,
+  handleFormChange,
+  formDataErrors,
+  setFormDataErrors,
+}: SellingOptionsSectionProps) {
+  const [currentOption, setCurrentOption] = useState<SellingOption>();
+  return (
+    <section className="my-10 flex flex-col">
+      <h4 className="text-lg font-bold">Choose Selling Option</h4>
+      <p className="text-sm mt-3">
+        Decide if you would like to only mint the NFT or list directly on
+        HashGuild. After minting, the NFT is accesible on your HashGuild
+        account.
+      </p>
+      <div className="my-8 flex gap-x-10 select-none">
+        {sellingOptions.map((option) => (
+          <div
+            aria-hidden="true"
+            key={option.index}
+            onClick={() => {
+              setCurrentOption(option);
+              setFormDataErrors({
+                ...formDataErrors,
+                sellingOptionError: false,
+              });
+              setFormData({ ...formData, sellingOption: option.name! });
+            }}
+            className={classNames(
+              'flex flex-col items-center justify-center gap-y-1 basis-1/2 py-6 border shadow-xl',
+              currentOption?.index === option.index ? 'border-black' : '',
+            )}
+          >
+            <option.icon />
+            <p>{option.name}</p>
+          </div>
+        ))}
+      </div>
+      {formDataErrors.sellingOptionError && (
+        <ErrorMessage errorText="Please select one option" />
+      )}
+
+      {/* <HbarLogo /> */}
+      {currentOption?.name === 'Mint and Sell' && (
```

Comment:
Disable the "Mint and Sell" option for now, this is a feature that will be added in the near future.

Reply:
done

Reply:
done